rmgpy/cantherm/molepro.py

```diff
@@ -42,7 +45,197 @@ class MoleProLog:
+                number[i] = 8
+                mass[i] = 15.99491461956
+            elif symbol[i] == 'P':
+                number[i] = 15
+                mass[i] = 30.97376163
+            elif symbol[i] == 'S':
+                number[i] = 16
+                mass[i] = 31.97207100
+            elif symbol[i] == 'Cl':
+                number[i] = 17
+                mass[i] = 35.4527
+            elif symbol[i] == 'I':
+                number[i] = 53
+                mass[i] = 126.90447
+            else:
+                print 'Atomic number {0:d} not yet supported in loadGeometry().'.format(number[i])
+        return coord, number, mass
+
+    def loadConformer(self, symmetry=None, spinMultiplicity=None, opticalIsomers=1, symfromlog=None):
+        """
+        Load the molecular degree of freedom data from a log file created as
+        the result of a MolPro "Freq" quantum chemistry calculation with the thermo printed.
+        """
+
+        modes = []
+        E0 = 0.0
+
+        f = open(self.path, 'r')
+        line = f.readline()
+        while line != '':
+
+            # The data we want is in the Thermochemistry section of the output
+            if 'THERMODYNAMICAL' in line:
+                modes = []
+                inPartitionFunctions = False
+                line = f.readline()
+                while line != '':
+
+                    # This marks the end of the thermochemistry section
+                    if '*************************************************' in line:
+                        break
+
+                    # Read molecular mass for external translational modes
+                    elif 'Molecular Mass:' in line:
+                        mass = float(line.split()[2])
+                        translation = IdealGasTranslation(mass=(mass,"amu"))
+                        modes.append(translation)
+                    # Read MolPro's estimate of the external symmetry number
+                    elif 'Rotational Symmetry factor' in line and symmetry is None:
+                        if symfromlog is True:
+                            symmetry = int(float(line.split()[3]))
+
+                    # Read moments of inertia for external rotational modes
+                    elif 'Rotational Constants' in line and line.split()[-1]=='[GHz]':
+                        inertia = [float(d) for d in line.split()[-4:-1]]
+                        for i in range(3):
+                            inertia[i] = constants.h / (8 * constants.pi * constants.pi * inertia[i] * 1e9) *constants.Na*1e23
+                        rotation = NonlinearRotor(inertia=(inertia,"amu*angstrom^2"), symmetry=symmetry)
+                        modes.append(rotation)
+
+                    elif 'Rotational Constant' in line and line.split()[3]=='[GHz]':
+                        inertia = float(line.split()[2])
+                        inertia[0] = constants.h / (8 * constants.pi * constants.pi * inertia[0] * 1e9) *constants.Na*1e23
```

Comment:
Since `inertia` is a float under this condition and not a list as above, we can't call it or assign into it as `inertia[0]`.
I think that all `inertia[0]` instances in this line should be replaced by `inertia`.